src/common/SurgeSynthesizer.cpp

```diff
@@ -3496,7 +3498,7 @@ bool SurgeSynthesizer::isBipolarModulation(modsources tms) const
     }
     if (tms == ms_keytrack || tms == ms_lowest_key || tms == ms_highest_key ||
         tms == ms_latest_key || tms == ms_pitchbend || tms == ms_random_bipolar ||
-        tms == ms_alternate_bipolar || tms == ms_timbre)
+        tms == ms_alternate_bipolar || (tms == ms_timbre && !mpeTimbreIsUnipolar))
```

Comment:
So this is the thing I'm not sure about.

TIMBRE is a combination of what we get from the keyboard and the note expression. The note expression is bipolar. So the modulation would still be potentially bipolar. Also for a given patch on a different device, it would be. I sort of feel like we should display the modulator as bipolar still..

This is just a ui thing, but thoughts welcome

Reply:
Hmm that's a valid point. Will remove.